hoot-services/src/main/java/hoot/services/controllers/osm/MapResource.java

```diff
@@ -579,50 +603,70 @@ public Response getMBR(@QueryParam("mapId") final String mapId) throws Exception
         Connection conn = DbUtils.createConnection();
         JSONObject ret = new JSONObject();
         try {
-            log.info("Retrieving map data for map with ID: " + mapId + " ...");
-
-            log.debug("Initializing database connection...");
+            log.info("Retrieving MBR for map with ID: " + mapId + " ...");
 
-            QMaps maps = QMaps.maps;
-            long mapIdNum = ModelDaoUtils.getRecordIdForInputString(mapId, conn, maps, maps.id, maps.displayName);
-            assert (mapIdNum != -1);
-
-            BoundingBox queryBounds = null;
+            long mapIdNum = -2;
             try {
-                queryBounds = new BoundingBox("-180,-90,180,90");
-                log.debug("Query bounds area: " + queryBounds.getArea());
+                mapIdNum = Long.parseLong(mapId);
             }
-            catch (Exception e) {
-                throw new Exception("Error parsing bounding box from bbox param: " + "-180,-90,180,90" + " ("
-                        + e.getMessage() + ")");
+            catch (NumberFormatException e) {
+                //
             }
+            if (mapIdNum == -1) // OSM API db
+            {
+                // OSM API database data can't be displayed on a hoot map, due to differences
+                // between the display code, so we arbitrarily returning roughly a CONUS bounds 
+                // here...not quite sure what else to return...
```

Comment:
I may be missing something, but does Hoot read in a clipped section of data from an osm api db before performing conflation?  Or does it read features in directly from the remote db as its comparing them to the features in the secondary dataset?